uni/lib/model/entities/time_utilities.dart

```diff
@@ -2,4 +2,14 @@ extension TimeString on DateTime {
   String toTimeHourMinString() {
     return '${hour.toString().padLeft(2, '0')}:${minute.toString().padLeft(2, '0')}';
   }
+
+  static const weekdays = [
+    'Segunda-Feira',
+    'Terça-Feira',
+    'Quarta-Feira',
+    'Quinta-Feira',
+    'Sexta-Feira',
+    'Sábado',
+    'Domingo'
+  ];
```

Comment:
You are extending DateTime to add new functionality. It's probably better to encapsulate this via something like `getWeekDaysStrings(bool startMonday = true, includeWeekend = true)`

Reply:
This makes sense, but i don't see the need for `startMonday` ?

Reply:
Just for clarity, since in the USA, the week usually starts Monday! Not a big deal.